docs/nginx/events.md

```diff
@@ -0,0 +1,54 @@
+Semantic conventions for Nginx Ingress Controller events
+
+**Status**: [Experimental](../../../document-status.md)
+
+<details>
+<summary>Table of Contents</summary>
+<!-- Re-generate TOC with `markdown-toc --no-first-h1 -i` -->
+
+<!-- toc -->
+
+- [Access](#access)
+- [Error](#error)
+
+<!-- tocstop -->
+
+</details>
+
+This document describes the semantic conventions for events in Nginx Ingress Controller. All Nginx Ingress Controller events MUST use a namespace of `nginx_ingress_controller` in the `event.name` property.
+
+## Access
+
+The event name MUST be `nginx_ingress_controller.access`.
+
+This event describes the details of an the Nginx access.
+
+| Body Field | Type | Description | Examples | Requirement Level |
+|---|---|---|---|---|
+| `http.request.id` | string | The randomly generated ID of the request | `529a007902362a5f51385a5fa7049884` | Required |
+| `http.request.length` | int | The request length (including request line, header, and request body) | `89` | Recommended |
```

Comment:
We should be using the Semantic Convention field [`http.request.size`](https://github.com/open-telemetry/semantic-conventions/blob/02ecf0c71e9fa74d09d81c48e04a132db2b7060b/model/registry/http.yaml#L109) for this.